iam.tf

```diff
@@ -125,3 +127,49 @@ resource "aws_iam_policy" "cloudwatch" {
   name   = "${module.label.id}-cloudwatch-execution"
   policy = data.aws_iam_policy_document.cloudwatch[0].json
 }
+
+data "aws_iam_policy_document" "cloudwatch_log_group_kms" {
```

Comment:
Add new kms key module variable `cloudwatch_log_group_kms_enabled` + attach condition to the `aws_iam_policy_document.cloudwatch_log_group_kms`